src/Leaderboard.py

```diff
@@ -83,7 +85,7 @@ def reportMatch(player: Member, whoWon: Team) -> str:
         team=foundPlayer[MatchKey.TEAM]
     )
     msg = reportConfirm(player, match, whoWon)
-    if (msg != ""):
+    if (not msg):
```

Comment:
I don't think `msg` makes sense here. That makes me think it's a string when actually it's a bool. I would rename it to something like `report_confirm_success` or something like that.

Reply:
Resolved.